src/main/java/com/nerodesk/om/mock/MkAttributes.java

```diff
@@ -89,11 +91,37 @@ public Date created() throws IOException {
 
     @Override
     public boolean visible() throws IOException {
-        return this.shown;
+        final boolean shown;
+        if (this.file.exists()) {
```

Comment:
@carlosmiranda I'm wondering, wouldn't it be better to skip `file.exists` check and let this method and also `show` just throw IOException if the file does not exist?  User can't change visibility if the file doesn't exist